lib/checkWipDraftPR.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ * @returns {Boolean}
+ */
+const isDraftPr = (pullRequest) => {
+  return pullRequest.draft;
+};
+
+/**
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ * @returns {Boolean}
+ */
+const isWIPPr = ({ title, body }) => {
+  return (
+    title.toLowerCase().includes('wip') || body.toLowerCase().includes('wip')
+  );
+};
+
+/**
+ * @param {import('probot').Context} context
+ */
+module.exports.checkWIP = async (context) => {
+  /**
+   * @type {import('probot').Octokit.PullsGetResponse} pullRequest
+   */
+  const pullRequest = context.payload.pull_request;
+  const prAuthor = pullRequest.user.login;
+
+  if (isDraftPr(pullRequest) || isWIPPr(pullRequest)) {
+    const link = 'here'.link(
+      'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia' +
+      '#wip--draft-pull-requests');
+    // Comment on Pull Request.
+    const commentBody = (
+      'Hi @' + prAuthor + ', WIP/Draft PRs are highly discouraged. You can ' +
+      'learn more about it ' + link + '. Do well to reopen it when it\'s ' +
```

Comment:
You can reopen it .... and ensure that it is without any WIP phrase in title or body.

Reply:
Done.